test/models/seat_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class SeatTest < ActiveSupport::TestCase
```

Comment:
If these files are not being used then please delete them.

Reply:
removed . thank you